internal/cli/atlas/dbusers/delete.go

```diff
@@ -54,11 +55,16 @@ func DeleteBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "delete <username>",
 		Aliases: []string{"rm"},
-		Short:   "Delete a database user for your project.",
+		Short:   "Remove the specified database user from your project.",
 		Args:    require.ExactArgs(1),
 		Annotations: map[string]string{
-			"usernameDesc": "Username to delete from the MongoDB database.",
+			"usernameDesc": "Username to delete from the MongoDB database. The format of the username depends on the user's method of authentication.",
```

Comment:
Question: Does this delete the user from the database or does this disassociate the user from the project? I am guessing the latter from the command and suggest the following: 
```suggestion
			"usernameDesc": "Username of user in the MongoDB database to delete from the specified project. The format of the username depends on the user's method of authentication.",
```

Reply:
I think that command would be atlas projects users delete and this does something different, but I will leave that for @gssbzn to answer

Reply:
yes, this is about clusters/deployments access and not project access if that makes it any clearer